README.md

```diff
@@ -4,17 +4,25 @@ The next-generation master server for Northstar.
 
 ## Installation
 
-First install GO from here: [https://go.dev/doc/install](https://go.dev/doc/install)
+First install Go from here: [https://go.dev/doc/install](https://go.dev/doc/install)
 
 Then run the following commands:
 
 ```bash
-go build cmd/atlas/main.go 
+go run cmd/atlas/main.go 
+```
+
+## Building
+
+To build Atlas, run the following command:
+
+```bash
+go build -o atlas.exe cmd/atlas/main.go
```

Comment:
`go run|build ./cmd/atlas`

Reply:
Running either of those you'll get `package cmd/atlas is not in std`

Reply:
Updated, I was using it without the `./` before so it failed whenever I tried it